cmd/http-stats.go

```diff
@@ -189,6 +222,14 @@ func (st *HTTPStats) updateStats(r *http.Request, w *httpResponseRecorder, durat
 			st.successDELETEs.Duration.Add(durationSecs)
 		}
 	}
+
+	if !strings.HasPrefix(r.URL.Path, "/minio") && !strings.HasSuffix(r.URL.Path, prometheusMetricsPath) {
```

Comment:
Use the constant for /minio

Reply:
You already set recordRequest with isS3 can we not reuse it here ?

Reply:
yes i will reuse this. Thanks